### DISCUSSION TITLE
How do I use this (Windows)? I downloaded this because the desktop authenticator isn't working.

### DISCUSSION THREAD

**Discussion by TheEaterOfBeans** [dyc3/steamguard-cli#332] [Q&A] [answered]
I opened it (following the instructions in the quickstart), but all that happened is I got a list of a bunch of commands and I don't know which ones I'm supposed to run and in what order.

Edit: Managed to find the setup. Still not sure if it's actually usable yet.

**Comment by dyc3** [comment, marked as answer]
Did you read this yet? https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md

**Reply by TheEaterOfBeans** [reply]
It only explains how to run this, not what all the commands do.

**Reply by dyc3** [reply]
Run `steamguard --help` to get the help text.
